www/js/crypter/crypto_app.js

```diff
@@ -165,9 +274,11 @@ window.filesender.crypto_app = function () {
             var blobArray = [];
 
             try {
-                this.generateKey(encryption_details, function (key) {
+                var chunkid = 0;
+                this.generateKey(chunkid, encryption_details, function (key) {
 		    var wrongPassword = false;
 		    var decryptLoop = function(i) {
+                        
```

Comment:
This may be the place to verify that all IV values share the same 96-bit random value to indicate that it belongs to the same file, as stated on Page 19:

  * The 96 bits worth of random material in the IV are produced from a secure random generator, and kept constant for one file (or one upload, as desired). This 96-bit value can be used to see if chunks belong to the same file or upload.

Reply:
This is also the place to verify that the counter portion rises as expected.